cookienotice/cookienotice.php

```diff
@@ -5,87 +5,66 @@
  * Description: Configure, show and handle a simple cookie notice
  * Version: 1.0
  * Author: Peter Liebetrau <https://socivitas/profile/peerteer>
- * 
  */
-use Friendica\Core\Hook;
+
+use Friendica\App;
 use Friendica\Core\Config;
+use Friendica\Core\Hook;
 use Friendica\Core\L10n;
 use Friendica\Core\Renderer;
 
 /**
  * cookienotice_install
  * registers hooks
- * 
+ *
  * @return void
  */
 function cookienotice_install()
 {
 	Hook::register('page_content_top', __FILE__, 'cookienotice_page_content_top');
 	Hook::register('page_end', __FILE__, 'cookienotice_page_end');
-	Hook::register('addon_settings', __FILE__, 'cookienotice_addon_settings');
```

Comment:
How are the settings now called if you remove the hook?

Reply:
Definitely.

Reply:
Done: https://github.com/friendica/friendica/pull/7122